source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -274,6 +274,41 @@ def prepend_path_item_with_access_point(path_items, stop_point, access_point):
         path_items[-1].CopyFrom(tmp_item)
 
 
+def prepend_path_item_with_poi_access(fallback_type, path_items, poi_access):
```

Comment:
the name is a bit misleading, you `append` in case of ENDING_FALLBACK and `prepend` in case of BEGINNING_FALLBACK, so I suggest to rename
```suggestion
def add_path_item_with_poi_access(fallback_type, path_items, poi_access):
```

Reply:
Done